### PR TITLE
Allow users to loan books

### PR DESCRIPTION
**Pull request by kwunkiuma**

### CODE REVIEW

**Comment by kwunkiuma** [comment]
image

**Comment by kwunkiuma** [comment]
For now loaning redirects the user to their account page, would it be preferred to show a confirmation page?

**Comment by FridaTveit** [comment]
So their account page would show this book as being on loan to them right? In that case I think it's fine. The most important thing is that it's obvious from the page we redirect them to that the borrowing was successful 🙂

File: Bookish.DataAccess/Services/BookishService.cs [hunk 1]
```diff
@@ -13,6 +14,7 @@ public interface IBookishService
         IEnumerable<BookCopy> GetCopies(string filter);
         IEnumerable<BookLoan> GetLoans(string userId);
         void AddBook(string title, string author, string isbn, int totalCopies);
+        void AddLoan(int copyId, string lenderId);
```
Comment (FridaTveit) [inline Bookish.DataAccess/Services/BookishService.cs]: `LoanBook`, `TakeOutBook` or `BorrowBook` might be more descriptive? 🙂

**Review by FridaTveit — COMMENTED** [review]
One minor thing, otherwise looks good 🙂

**Review by FridaTveit — APPROVED** [review]
Looks great 🙂

**Pull request merged**